### DISCUSSION TITLE
✅ Quickview button on product archive widget.

### DISCUSSION THREAD

**Discussion by sayeedbaksh** [elementor/elementor#24552] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I use Astra Pro and another widget from Xploded Theme to achieve a responsive popup option. 
Popup are essential for ecommerce stores. 

### Describe the solution you'd like

To have a popup button and window for the product widget that is super responsive. the button must be visible at all time on a mobile view. 

### Describe alternatives you've considered

I had to use this plugin with CSS to position the button the way i like. 
Positioning the button is critical for design.
https://demos.xplodedthemes.com/woo-quick-view/
I use astra pro and their popup is slow. 

### Additional context

If elementor sold an ecommerce addon that has a stylish popup login page, refine options for ecommerce then we don't need astra. keep everything elementor. more plugins, more issues.

**Comment by nicholaszein** [comment]
Hi!

This feature request was solved with the release of Elementor Pro version 3.26.0.

You can now use the Nested Off-Canvas widget inside the Loop Item template to create your own Quickviews for WooCommerce products, and display them in the Loop Grid.

More info:
https://elementor.com/help/add-an-off-canvas-widget-to-a-loop-grid/

Regards
